release-notes.txt

```diff
@@ -0,0 +1,208 @@
+BISDN Linux
+v4.6.0 Release Notes
+Date: 2022-05-03
+
+! New in this release
+- New platform Accton-AS5835-54x is now supported
+- baseboxd: bump version to 1.11.2
+- Licenses in build process were updated to use the correct SPDX identifier
+- Linux kernel updated to version 5.15.35
+- OF-DPA updated to support configuraton of MPLS pseudowires using RYU
+- FRR updated to version 8.2
+- baseboxd/OF-DPA: support added for creating and enabling Broadcom KNET interfaces
+
+! Bug fixes
+- sysctl max read buffer size was set lower than the defaul read buffer size, causing baseboxd to intitialize with a buffer size that is too small. This could lead to loss of netlink messages, causing an out of sync switch state
```

Comment:
Seems a bit long. How about:
```suggestion
- increase max read buffer to avoid loss of netlink messages (and out of sync switch state) under heavy load
```